### PR TITLE
chore(viewer): add h6 to the available HTML tags

### LINKED ISSUE #412: H6 header disappears in markdown text view
__Describe the Bug__

We don't have the equivalent of an h6 with `######`—the header just disappears.

__Steps to Reproduce__

Create a text view with a h6 header—and it will disappear:

Bildschirmfoto 2022-11-16 um 16 32 35

__Expected Behavior__

Show that header

__Environment__

- Latest playground as of Nov 16

### PR DESCRIPTION
**Pull request by adrianmarinwork**
Closes #412

### CODE REVIEW

**Review by pinussilvestrus — APPROVED** [review]
Looks good to me! Thank you very much 🎉

File: packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js [hunk 1]
```diff
@@ -46,6 +46,8 @@ describe('Text', function() {
 ## h2
 ### h3
 #### h4
+##### h5
+###### h6
```
Comment (pinussilvestrus) [inline packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js:50]: We need to adjust the `expect` part of this test as well: https://github.com/bpmn-io/form-js/pull/450/files#diff-f782522d93784395284f1b8e2c8b29c8ff07908fcb77815c9f527d9aa8222e0dR83

You can execute the tests locally as well via `npm run test` or `npm run dev -- '@bpmn-io/form-js-viewer`

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
I was a bit too fast and didn't wait for the CI 😅 Let's fix the test and merge afterward!

**Comment by adrianmarinwork** [comment]
Okay, I've updated the expected value of the test with the new HTML tags!

**Comment by pinussilvestrus** [comment]
Works now, thanks! 👍

**Pull request merged**